.github/ISSUE_TEMPLATE/lit-element-next-bug-report.md

```diff
@@ -0,0 +1,45 @@
+name: LitElement lit-next preview bug report
+about: Use this template to report an issue with the LitElement PREVIEW release (lit-element@next-major)
+title: '[lit-element] '
+labels: 'lit-next'
+assignees: ''
+
+---
+
+<!--
+Please read our contribution guidelines:
+  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
```

Comment:
Do we want to include some text about searching for known issues, like in the new READMEs and CHANGELOGs?

Reply:
I can't find that text in the new READMEs and CHANGELOGs, only in the blog post, FWIW. But will add it here.